src/Native/HtmlAsJson.js

```diff
@@ -1,8 +1,17 @@
 var _eeue56$elm_html_test$Native_HtmlAsJson = (function() {
+    function forceThunks(key, vNode) {
+        if (typeof vNode !== 'undefined' && vNode.type === 'thunk' && !vNode.node) {
+            vNode.node = vNode.thunk.apply(vNode.thunk, vNode.args);
+            return vNode;
+        } else {
+            return vNode;
+        }
+    }
+
     // stringify needed to strip functions - can performance-optimize this later!
     return {
         toJsonString: function(html) {
-            var asString = JSON.stringify(html);
+            var asString = JSON.stringify(html, forceThunks);
```

Comment:
🎨 would rather do this as a separate step rather than using the stringifier function